projects/midnight-commander.org/package.yml

```diff
@@ -54,5 +54,5 @@ build:
 
 # FIXME: hard to test interactive tools
 test:
-  test "$(mc --datadir | cut -d' ' -f1)" = "{{prefix}}/etc/mc/"
-  # ^^ tests relocatability
+  test "$(mc --datadir | cut -wf1)" = "{{prefix}}/etc/mc/"
```

Comment:
linux cut doesn't have -w

Reply:
And --datadir can return an array. Hence: `test "$(mc --datadir | cut -d ' ' -f1 | cut -d: -f1)" = "{{prefix}}/etc/mc/"`